iscript/tests/test_mac.py

```diff
@@ -1233,19 +1232,9 @@ def fake_get_notarization_status_from_log(path):
             ]
         }
     }
-    if use_langpack:
-        mocker.patch.object(mac, "sign_langpacks", new=noop_async)
-        task["payload"]["upstreamArtifacts"].append(
-            {
-                "taskId": "task3",
-                "formats": ["autograph_langpack"],
-                "paths": ["public/build3/target.langpack.xpi"],
-            }
-        )
```

Comment:
we probably have to mock the notarization calls to get these tests to pass. we also need to make sure this works irl.